index.html

```diff
@@ -5071,40 +5071,87 @@ <h2 id="AudioWorkerParamDescriptor">
-          supporting custom AudioNodes. This processing mechanism ensures the
-          synchronous execution of the script code with other built-in
-          <code>AudioNode</code>s in the audio graph.
-        </p>
-        <p>
-          The <code>AudioWorklet</code> object stores the definitions of custom
-          audio processors and <code>AudioWorkletNode</code>s can be
-          constructed based on these definitions. The registered definition in
-          an AudioWorklet is valid for the entire global scope (i.e.
-          <code>window</code>), therefore any <a><code>AudioContext</code></a>
-          instance in the same global scope can access the registered
-          definition to create a custom node.
-        </p>
-        <p>
-          After importing a module script into the
-          <a><code>AudioWorkletGlobalScope</code></a>, an instance of
-          <a><code>AudioWorkletNode</code></a> can be created in the main
-          global scope, based on any of the definitions in the script. At the
-          same time, the user agent automatically creates an instance of
-          <a><code>AudioWorkletProcessor</code></a>. This counterpart
-          represents an audio processing module that runs on the audio
-          <a>rendering thread</a>. These two components can communicate via
-          message passing.
-        </p>
+        <section>
+          <h2 id="AudioWorklet-concepts">
+            Concepts
+          </h2>
+          <p>
+            The <code>AudioWorklet</code> object allows developers to supply
+            JavaScript to process audio on the <a>rendering thread</a>,
+            supporting custom AudioNodes. This processing mechanism ensures the
+            synchronous execution of the script code with other built-in
+            <code>AudioNode</code>s in the audio graph.
+          </p>
+          <p>
+            An associated pair of objects must be defined in order to realize
+            this mechanism; <code>AudioWorkletNode</code> and
+            <code>AudioWorkletProcessor</code>. The former represents the
+            interface for the main global scope similar to other AudioNode
+            objects and the latter implements the internal audio processing
+            within a special scope named <code>AudioWorkletGlobaScope</code>.
+          </p>
+          <figure>
+            <img alt="AudioWorklet concept" src=
+            "images/audioworklet-concept.png">
+            <figcaption>
+              <code>AudioWorkletNode</code> and
+              <code>AudioWorkletProcessor</code>
+            </figcaption>
+          </figure>
+          <blockquote>
+            <pre>// In the main global scope
+window.audioWorklet.import("bypass.js").then(function () {
+  var context = new AudioContext();
+  var bypass = new AudioWorkletNode(context, "Bypass");
+});</pre>
+          </blockquote>
+          <p>
+            An <code>AudioWorkletNode</code> and its corresponding processor
```

Comment:
I thought that only the Processor could be registered in the imported file. The Node itself is never actually referenced in the imported JS since it's all within the AWGS.